docker-for-mac/release-notes.md

```diff
@@ -16,10 +16,22 @@ This page contains information about the new features, improvements, known issue
 > Starting with Docker Desktop 3.0.0, Stable and Edge releases are combined into a single release stream for all users. Updates to Docker Desktop will now be available automatically as delta updates from the previous version. This means, when there is a newer version of Docker Desktop, it will be automatically downloaded to your machine. All you need to do is to click **Update and restart** from the Docker menu to install the latest update.
 {: .important }
 
+## Docker Desktop 3.2.2
+2021-03-15
+
+> [Download](https://desktop.docker.com/mac/stable/amd64/Docker.dmg)
+
+### Bug fixes and minor changes
+
+- Fixed an issue that stopped containers binding to port 53. Fixes [docker/for-mac#5416](https://github.com/docker/for-mac/issues/5416).
+- Fixed an issue running 32-bit Intel binaries in emulation on Intel CPUs. Fixes [docker/for-win#10594](https://github.com/docker/for-win/issues/10594).
```

Comment:
This makes it sound like they should run in emulation but it was broken. Change to:
"Fixed an issue that 32-bit Intel binaries were emulated on Intel CPUs"

Reply:
Updated in both files.